hack/run.sh

```diff
@@ -54,6 +55,8 @@ elif [[ "${action}" == "deploy-source" ]]; then
 	source "${ROOT_DIR}"/test/e2e-common.sh && test_source_setup
 elif [[ "${action}" == "build-from-source" ]]; then
 	source "${ROOT_DIR}"/test/e2e-common.sh && build_components_from_source
+elif [[ "${action}" == "deploy-control-plane" ]]; then
+	source "${ROOT_DIR}"/test/e2e-common.sh && build_control_plane_from_source
```

Comment:
@Sanskarzz you still need to call the install_control_plane_from_source function here as well, after build_control_plane_from_source